src/main/java/digital/slovensko/autogram/core/errors/MultipleOriginalDocumentsFoundException.java

```diff
@@ -1,15 +1,8 @@
 package digital.slovensko.autogram.core.errors;
 
-public class MultipleOriginalDocumentsFoundException extends RuntimeException {
+public class MultipleOriginalDocumentsFoundException extends AutogramException {
 
     public MultipleOriginalDocumentsFoundException(String description) {
-        super("Multiple original documents found");
-        this.description = description;
-    }
-
-    private String description;
-
-    public String getDescription() {
-        return description;
+        super("Chyba ASiC-E kontajnera", "Nájdených viacero dokumnetov na podpis", description);
```

Comment:
Toto je user-facing? Treba aj povedat, ze co s tym?

Reply:
Ako to teda vyzera teraz?

Reply:
Teraz sa ukáže rovno toto:
![image](https://github.com/slovensko-digital/autogram/assets/12500066/6d828c93-bb65-4182-9cbf-6123e524110d)

Na tej predošlej obrazovke ešte musel user kliknúť na "Pokračovať".